src/parser/whamm_parser.rs

```diff
@@ -401,6 +401,84 @@ fn stmt_from_rule(pair: Pair<Rule>, err: &mut ErrorGen) -> Result<Statement, Vec
                 }
             }
         }
+        Rule::incrementor => {
+            trace!("Entering incrementor");
+            let mut pair = pair.into_inner();
+            let var_id_rule = pair.next().unwrap();
+            let var_id_line_col = LineColLocation::from(var_id_rule.as_span());
+            let var_id = Expr::VarId {
+                name: var_id_rule.as_str().parse().unwrap(),
+                loc: Some(Location {
+                    line_col: var_id_line_col.clone(),
+                    path: None,
+                }),
+            };
+            let _val = Value::Integer {
+                ty: DataType::I32,
+                val: 1,
+            };
+            let _rhs = Expr::Primitive {
```

Comment:
Prepending an underscore on a var name implies that the var isn't being used. I'd refactor to remove the underscores.